cloudformation/calculation-athena.yaml

```diff
@@ -201,6 +201,37 @@ Resources:
 
         Workgroup: !Ref AthenaTransactionWorkgroup
 
+  BillingAndTransactionsCuratedView:
+    Type: Custom::AthenaView
+    DependsOn: TransactionsCuratedView
+    Properties:
+      ServiceToken: !GetAtt CustomAthenaViewResourceFunction.Arn
+      View:
+        Database: !Ref CalculationsDB
+        Name: btm_billing_and_transactions_curated
+        Query: |
+          SELECT 
+              b.vendor_name,
+              b.service_name,
+              COALESCE(b.year,t.year)                                                                                                         AS year,
+              COALESCE(b.month,t.month)                                                                                                       AS month,
+              GREATEST(COALESCE(b.price,0),COALESCE(t.price,0))-LEAST(COALESCE(b.price,0),COALESCE(t.price,0))                                AS price_difference,
+              GREATEST(COALESCE(b.quantity,0),COALESCE(t.quantity,0))-LEAST(COALESCE(b.quantity,0),COALESCE(t.quantity,0))                    AS quantity_difference,
```

Comment:
Yeah I think we'll need to keep hold of the sign, presumably the dashboard will need to indicate if the invoice was more than the transactions or the other way around

Reply:
I agree, this difference should be signed -- this will simplify this statement to just `COALESCE(invoice.price,0)-COALESCE(txn.price,0` as well.

Reply:
The newest commit has changes to implement the formula as originally specified, which will allow the differences and percentage differences to go negative.  In the case where there are 0 transactions registered, we return null to avoid division by zero, as Athena will not return Infinity or NaN or anything by default -- it just fails the whole query, which is not desired behaviour.

Reply:
This is in there now.